DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/SetDelegatePasswordController.cs

```diff
@@ -68,14 +63,9 @@ bool isFromViewDelegatePage
                 return View(model);
             }
 
-            var delegateUser = userDataService.GetDelegateUserById(delegateId)!;
-
-            if (string.IsNullOrWhiteSpace(delegateUser.EmailAddress))
-            {
-                return View("NoEmail");
-            }

```

Comment:
I think this is a question for @SteveJacksonSoft

Reply:
So this is for the forgot/reset-password journey only right?*
* The ForgotPassword POST endpoint validates that the email provided is valid.
* The user won't know their GUID until after they log in, so it's very unlikely they'll pass that in as their email, so we're only worried about people with invalid emails pre-uar. So should we show a new view for them?
* These users will only have a sudden need to reset their emails post-uar if there are multiple delegates with that "email" and different passwords. Otherwise they've been in trouble since forever.
* If they do, they fall under the larger class of "Delegates with same email and different passwords who can't access their email". It would make sense to add a line to the "Please reset your password" login page to say "If you can't access your email, please contact ..."

So I'll take that idea and run with it. Nothing to change on this ticket I don't think.

*Since we don't send emails in the change-your-own-password or the change-a-delegate's-password journeys.